sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/rule/GatewayFlowRule.java

```diff
@@ -52,6 +53,23 @@ public GatewayFlowRule(String resource) {
         this.resource = resource;
     }
 
+    public long calIntervalSec() {
```

Comment:
Ditto. This logic can be moved to the dashboard.

Reply:
It has been removed.